crates/pop-cli/README.md

```diff
@@ -0,0 +1,258 @@
+# Pop CLI
+
+<img src=".icons/logo.jpeg"></img>
+
+An all-in-one tool for Polkadot development.
+
+## Install
+
+You can install Pop CLI as follows:
+
+```shell
+cargo install --locked --git https://github.com/r0gue-io/pop-cli
+```
+
+> :information_source: A [crates.io](https://crates.io/crates/pop-cli) version will be available soon!
+
+## Getting Started
+
+### Parachains
+
+Use `pop` to generate a new parachain from a template:
+
+```sh
+# Create a minimal parachain
+pop new parachain my-app
+# Get a pallet-contracts enabled parachain
+pop new parachain my-app cpt
+# Get a evm compatible parachain
+pop new parachain my-app fpt
+```
+
+Use `pop` to build your parachain.
+
+```sh
+# Build your parachain
+pop build parachain -p ./my-app
+```
+
+or
+
+```sh
+cd my-app
+pop build parachain
+```
+
+You can also customize your parachain by providing config options for token symbol (as it appears in chain metadata),
```

Comment:
I suggest removing this bit until "Create a new pallet". 
- It distracts from the core functionalities
- There is already a lot of text in the README
- It is at the start of the README. Someone who wants to learn about all the capabilities of pop cli is not interested in this (I think)

Can be reintroduced when we make a more expanded document.

Reply:
Or replace them under `pop new parachain` above and introduce them in short like done in `pop new pallet` (with options)

Reply:
Good feedback, moved the pallet instructions into its own Pallets section.